### ISSUE TITLE
Add filter support for pull requests.

### ISSUE THREAD

**Issue by NicolasBonduel** [zacherkkila/GitHub-Default-Issue-Filter#2]
Hello,

It would be great if we could have the same thing for filtering PRs.

Thanks for your work!

**Comment by zacherkkila** [comment]
Yeah that is a good idea, not sure when I will get to it here but I will add that in

**Comment by sancarn** [comment]
Would be really useful for `-label:dependencies` to get rid of all the dependabot bumps.
